### PR TITLE
fixed excessive whitespace around buttons

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* None

*Description of changes:*
While investigating design options with Scott, we found that the Todo List buttons had excessive whitespace around them (compared with deployed code in Test and Prod). This PR fixes that.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: coops-ui/src/components/Dashboard/TodoList.vue [hunk 1]
```diff
@@ -109,90 +109,88 @@
...
+                      <v-list-item-title>Delete Draft</v-list-item-title>
+                    </v-list-item>
+                  </v-list>
+                </v-menu>
+              </template>
+
+                <template v-else-if="isPending(item)">
+                <v-btn class="btn-resume-payment"
                   color="primary"
-                  :disabled="!item.enabled || coaPending || !confirmCheckbox"
```
Comment (cameron-freshworks) [inline coops-ui/src/components/Dashboard/TodoList.vue]: Have you refactored how the button validates there isn't a `coaPending` or the `confirmCheckbox` isn't checked. I just don't see that code in this PR so i'm a little confused.

File: coops-ui/src/components/Dashboard/TodoList.vue [hunk 2]
```diff
@@ -109,90 +109,88 @@
...
+              <template v-if="inProcessFiling !== undefined && inProcessFiling === item.id">
+                <v-btn text loading disabled />
+              </template>
+
+              <template v-else-if="isDraft(item)">
+                <v-btn class="btn-draft-resume"
                   color="primary"
                   :disabled="!item.enabled"
-                  @click.native.stop="doResumePayment(item)"
+                  @click.native.stop="doResumeFiling(item)"
```
Comment (cameron-freshworks) [inline coops-ui/src/components/Dashboard/TodoList.vue]: I don't see a refactor of the method itself in this file, maybe i'm missing something.

File: coops-ui/src/components/Dashboard/TodoList.vue [hunk 3]
```diff
@@ -109,90 +109,88 @@
             </div>
 
             <div class="list-item__actions">
-              <v-col>
```
Comment (cameron-freshworks) [inline coops-ui/src/components/Dashboard/TodoList.vue]: Does the Annual report due date ( BCOMPs exclusive ) line up with the buttons after removing the `<v-col>`?

Comment (severinbeauvais) [reply coops-ui/src/components/Dashboard/TodoList.vue]: I'll check that...

File: coops-ui/src/components/Dashboard/TodoList.vue (hunk 1 above)
Comment (severinbeauvais) [reply coops-ui/src/components/Dashboard/TodoList.vue]: Argh, diff is playing tricks on us. On the Filed Changed page, click on the gear icon and ignore whitespace changes. You should see only 2 lines deleted.

File: coops-ui/src/components/Dashboard/TodoList.vue (hunk 2 above)
Comment (severinbeauvais) [reply coops-ui/src/components/Dashboard/TodoList.vue]: This wasn't refactored. See comment above.

File: coops-ui/src/components/Dashboard/TodoList.vue (hunk 1 above)
Comment (cameron-freshworks) [reply coops-ui/src/components/Dashboard/TodoList.vue]: Ahhh gotcha. That caught me off guard, i haven't had to check that before. Consider this resolved!

File: coops-ui/src/components/Dashboard/TodoList.vue (hunk 2 above)
Comment (cameron-freshworks) [reply coops-ui/src/components/Dashboard/TodoList.vue]: Same as above, ignore my comment!

File: coops-ui/src/components/Dashboard/TodoList.vue (hunk 3 above)
Comment (severinbeauvais) [reply coops-ui/src/components/Dashboard/TodoList.vue]: Yes, I think I broke that. I will re-engineer this so the Next AR Date displays above the File Now button. Stand by...

Comment (severinbeauvais) [reply coops-ui/src/components/Dashboard/TodoList.vue]: Fixed. I replaced the incorrect use of v-col, which came with the additional padding, with a div with the same "width:100%" property.

Comment (cameron-freshworks) [reply coops-ui/src/components/Dashboard/TodoList.vue]: Sounds good!

**Pull request merged**